contrib/helm/Makefile

```diff
@@ -0,0 +1,44 @@
+# create kind cluster with nginx-ingress as the most popular ingress controller for K8S
+.PHONY: deploy
+create-cluster:
+	kind create cluster --name oauth2-proxy --config kind-cluster.yaml
+	make setup-dns
+	make setup-ingress
+
+.PHONY: setup-ingress
+setup-ingress:
+	kubectl apply -f https://raw.githubusercontent.com/kubernetes/ingress-nginx/master/deploy/static/provider/kind/deploy.yaml
+	sleep 5 # because of https://github.com/kubernetes/kubernetes/issues/83242
+	kubectl --namespace ingress-nginx wait --for=condition=available --timeout=30s deployment/ingress-nginx-controller
+	kubectl --namespace ingress-nginx rollout status --timeout 90s deployment/ingress-nginx-controller
+
+# default Pod CIDR is 10.244.0.0/16 https://github.com/kubernetes-sigs/kind/blob/master/pkg/apis/config/v1alpha4/default.go
```

Comment:
Again, maybe permalink rather than pinning to master as this will be liable to change?